imap_processing/tests/lo/test_science_counts.py

```diff
@@ -0,0 +1,43 @@
+from random import choice
+
+import pytest
+
+from imap_processing.lo.l0.data_classes.science_counts import ScienceCounts
+
+
+# Going to wait until validation data is available to test that
+# the values are correct. Currently only check for the shape
+# of the resulting data.
+@pytest.mark.skip(reason="no data to initialize with")
+def test_science_counts():
+    """Test the science counts parsing, decompression, and shaping."""
+    ## Arrange
+    sc = ScienceCounts("fake_packet", "version", "packet_name")
+    sc.SCI_CNT = "".join(choice("01") for _ in range(26880))  # noqa: S311
```

Comment:
I'd avoid any random we can, especially if you're just going to skip it for now. Perhaps just use 01 for now (or all 0s)? (Same for the other test file)
```suggestion
    sc.SCI_CNT = "01" * 14440
```